source/administration/config-settings.rst

```diff
@@ -3995,12 +3995,12 @@ Note that even if ``town-square`` is not listed, every user is added to that cha
 | This feature's ``config.json`` setting is ``"ExperimentalDefaultChannels": []`` with string array input consisting of channel names, such as ``["announcement", "developers"]``. |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-Allow Users to View Archived Channels (Experimental)
+Allow Users to View Archived Channels (Beta)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**True**: Allows users to view permalinks and search for content of channels that have been archived. Users can only view the content in channels of which they were a member before the channel was archived.
+**True**: Allows users to view, share and search for content of channels that have been archived. Users can only view the content in channels of which they were a member before the channel was archived.
```

Comment:
Thanks @jasonblais, can you help me understand what "share" means in the context of archived channels? It's not immediately obvious to me why it needs to be called out specifically or what features promote the use of the word

Reply:
It was in relation to permalinks (share permalinks to archived channels), but maybe that's not necessary or can be worded better.